src/drc/api/auth.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from zds_schema.models import APICredential
+
+logger = logging.getLogger(__name__)
+
+
+def get_ztc_auth(url: str) -> dict:
```

Comment:
Waarom zitten deze functies ook niet in zds common?

Reply:
Was nog niet zeker hoe generiek ze (zouden) zijn.